deployment/Procfile

```diff
@@ -0,0 +1,6 @@
+manager-web-server: NTERNAL_PORT=MANAGER_INTERNAL_PORT EXTERNAL_PORT=$MANAGER_EXTERNAL_PORT WEBSOCKET_PORT=$MANAGER_WEBSOCKET_PORT npm --prefix services/manager start
```

Comment:
`NTERNAL_PORT`

I believe this is called doing a Dan.

Reply:
I think some of them might be missing $ signs

Reply:
Goodness me, how did that happen? Fixed.